org-roam.el

```diff
@@ -1447,7 +1530,8 @@ When NEW-FILE-OR-DIR is a directory, we use it to compute the new file path."
                              new-file
                            (car file)))
               (org-roam-with-file file nil
-                (org-roam--replace-link old-file new-file)
+                (org-roam--replace-link (list :path old-file :id file-id)
```

Comment:
We still need to figure out how to pass the title change over.

Reply:
title change will be passed down when `org-roam--update-links-on-title-change` is called. Not efficient to update each affected file potentially two times but it works. 
Maybe you are thinking changing the file name directly should also update the title of a file which then propagates to file name to backlinks along with the file title?

Reply:
removed `:id file-id`